charts/trino/values.yaml

```diff
@@ -125,10 +125,26 @@ initContainers: {}
   #     image: busybox:1.28
   #     command: ['sh', '-c', 'echo The worker is running! && sleep 3600']
 
+sidecarContainers: {}
+#   coordinator:
+#     - name: side-coordinator
+#       image: busybox:1.28
```

Comment:
is busybox really a good example?

Reply:
It is just an example - it depends on specific use case what sidecar's image will be used.